neqo-transport/src/connection.rs

```diff
@@ -1178,17 +1182,21 @@ impl Connection {
         let mut needs_padding = false;
 
         // Check whether we are sending packets in PTO mode.
-        let (pto, cong_avail, min_pn_space) =
+        let (pto, cong_avail, min_pn_space, only_not_cc) =
```

Comment:
I'm having real trouble with the name of this variable.  How about `cc_limited` if the goal is to not send anything but ACK when you are CC limited.  Comment to that effect on the check above.

Reply:
cc_limited won against bypass_cc.